csrc/ir/nodes.cpp

```diff
@@ -2891,10 +2891,10 @@ std::string IterDomain::toString(int indent_size) const {
   if (stop() != extent()) {
     ss << stop()->toInlineString() << " : ";
   }
-  if (isBroadcast() && hasExpandedExtent()) {
-    ss << expandedExtent()->toInlineString();
-  } else {
-    ss << extent()->toInlineString();
+  // TODO: before merging, undo this debug style or hide behind an option
+  ss << extent()->toInlineString();
+  if (hasExpandedExtent()) {
+    ss << " expanded to " << expandedExtent()->toInlineString();
```

Comment:
Changes `toString()` to look like
```
%kernel {                                                                                                                                
T4_l[ iS14{( i0 * i2 )}rf ] = view( T0_g[ iS0{i0}, iS1{i2} ] )      
T5_l[ iS15{( i0 * i2 )}, bS16{1} ]                                                                                                       
   = broadcast( T4_l[ iS14{( i0 * i2 )}rf ] )                       
T2_l[ iS18{i5}rf, bS19{1 expanded to i6}rf ] = expand( T5_l[ iS15{( i0 * i2 )}, bS16{1} ], {i5, i6} )
T3_g[ iS20{i5}, bS21{1 expanded to i6} ]                            
   = T2_l[ iS18{i5}rf, bS19{1 expanded to i6}rf ]                   
   + T2_l[ iS18{i5}rf, bS19{1 expanded to i6}rf ];    
```
This was useful for debugging since I was seeing expanded extents alongside non-unity regular extents. I have seen that before in other issues and it can be confusing. I am still leaning toward reverting this back to the existing behavior but I figured I'd leave it up for debate for now.
I was planning to res

Reply:
nitpick: `expanded to` feels a bit too elaborate, maybe simpler form as `->` or some simpler mark.

Reply:
Great point. Changed to `ex`, e.g.
```
%kernel {                                                                                                                                
T4_l[ iS14{( i0 * i2 )}rf ] = view( T0_g[ iS0{i0}, iS1{i2} ] )      
T5_l[ iS15{( i0 * i2 )}, bS16{1} ]                                                                                                       
   = broadcast( T4_l[ iS14{( i0 * i2 )}rf ] )                       
T2_l[ iS18{i5}rf, bS19{1 ex i6}rf ] = expand( T5_l[ iS15{( i0 * i2 )}, bS16{1} ], {i5, i6} )
T3_g[ iS20{i5}, bS21{1 ex i6} ]                            
   = T2_l[ iS18{i5}rf, bS19{1 ex i6}rf ]                   
   + T2_l[ iS18{i5}rf, bS19{1 ex i6}rf ]; 
```